src/api/apiConstants.ts

```diff
@@ -13,10 +13,10 @@ function timeConverter (unixTimestamp): string {
   const hour = addZero(a.getHours())
   const min = addZero(a.getMinutes())
   const sec = addZero(a.getSeconds())
-  return date + ' ' + month + ' ' + year + ' ' + hour + ':' + min + ':' + sec
+  return `${date} ${month} ${year} from ${hour}:${min}:${sec} IST.`
 }
```

Comment:
I think the timezone isn't in IST. Check and confirm once.

Reply:
Yes, it is in IST, checked and confirmed.